wafer/talks/models.py

```diff
@@ -112,8 +112,10 @@ class Meta:
     )
 
     talk_id = models.AutoField(primary_key=True)
-    talk_type = models.ForeignKey(TalkType, null=True, blank=True)
-    track = models.ForeignKey(Track, null=True, blank=True)
+    talk_type = models.ForeignKey(
+        TalkType, null=True, blank=True, on_delete=models.CASCADE)
+    track = models.ForeignKey(
+        Track, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
I was thinking the same but thought I should initially set them to exactly what they are now. Happy to change them to `SET_NULL` since they're allowed to be null currently anyway?